specifyweb/frontend/js_src/lib/components/QueryBuilder/CreateRecordSet.tsx

```diff
@@ -38,14 +37,19 @@ export function CreateRecordSet({
     | State<'Saving'>
   >({ type: 'Main' });
 
+  const resolvedRecordIds = React.useMemo(
+    () => (typeof recordIds === 'function' ? recordIds() : recordIds),
+    [recordIds]
```

Comment:
the getIds function provided as a prop to CreateRecordSetFromQuery is re-created on each render, so this useMemo would be called on each render
is that an issue?